src/Carousel.css

```diff
@@ -3,5 +3,33 @@ ul {
   list-style-type: none;
   padding: 0;
   margin: 0;
+  justify-content: center;
 }
 
+
+.carousel {
+  max-width:60%;
+  padding:50px 50px 50px 50px;
+  position:relative;
+  overflow: hidden;
+}
+
+.button {
+  background: none;
+  border: none;
+}
+
+.fa-angle-left, .fa-angle-right {
+  color: #af000e;
+  font-size: 50px;
+  transition: 1s;
```

Comment:
I like this transition, but it does feel a little long. Maybe if we cut the time down to .3-.7 it would feel more natural